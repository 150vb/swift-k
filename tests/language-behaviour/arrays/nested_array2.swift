type file;

app (file o) echo (string input)
{
  echo input stdout=@o;
}

main()
{
  foreach i in [0:0] {
    int ia[];
    foreach j in [0:1] {
      if (j==0) {
        ia[j] = 100;
      }
      else {
        ia[j] = 101;
      }
    }
    sub(ia);
  }
}

sub(int iap[])
{
  file f<"nested_array2.out">;
  f = echo(@sprintf("in sub: iap=%q",iap));
}

main();
