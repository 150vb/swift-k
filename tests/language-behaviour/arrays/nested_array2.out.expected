in sub: iap=[100,101]
